### PR TITLE
CEDS-6208

### PR DESCRIPTION
**Pull request by glidester**
Updated the HTML partials (again!) with a new message from the DC team that has better translations

### CODE REVIEW

File: app/uk/gov/hmrc/customs/declarations/stub/controllers/SecureTwoWayMessagingController.scala [hunk 1]
```diff
@@ -370,21 +368,21 @@ object SecureTwoWayMessagingController {
         |   }
         |</style>
         |<div class="govuk-body-l">
-        |   <h1 class="govuk-heading-l margin-top-small margin-bottom-small">about import doc test</h1>
+        |   <h1 class="govuk-heading-l margin-top-small margin-bottom-small">MRN123456789098765 needs some action</h1>
         |   <p>
         |      <span aria-hidden="true" class="custom-caption">
-        |      HMRC Imports(CDS) needs actions sent this on 17 May 2023 at 10:43am
+        |      National Clerence Hub sent this on 3 December 2024 at 3:00pm
```
Comment (IanWood-HMRC) [inline app/uk/gov/hmrc/customs/declarations/stub/controllers/SecureTwoWayMessagingController.scala:374]: Should it not be 
**National Clearance**

File: app/uk/gov/hmrc/customs/declarations/stub/controllers/SecureTwoWayMessagingController.scala [hunk 2]
```diff
@@ -451,21 +447,21 @@ object SecureTwoWayMessagingController {
         |         </div>
         |      </div>
         |   </div>
-        |   <h1 class="govuk-heading-l margin-top-small margin-bottom-small">about import doc</h1>
+        |   <h1 class="govuk-heading-l margin-top-small margin-bottom-small">MRN123456789098765 needs some action</h1>
```
Comment (IanWood-HMRC) [inline app/uk/gov/hmrc/customs/declarations/stub/controllers/SecureTwoWayMessagingController.scala:450]: This seems an English message, but Welsh times

Comment (glidester) [reply app/uk/gov/hmrc/customs/declarations/stub/controllers/SecureTwoWayMessagingController.scala:450]: This is user generated content so can not be translated

File: app/uk/gov/hmrc/customs/declarations/stub/controllers/SecureTwoWayMessagingController.scala (hunk 1 above)
Comment (glidester) [reply app/uk/gov/hmrc/customs/declarations/stub/controllers/SecureTwoWayMessagingController.scala]: Yes. Copied from the DC service in QA. Will update our copy ;)

**Review by IanWood-HMRC — APPROVED** [review]

**Pull request merged**